index.js

```diff
@@ -144,9 +144,14 @@ HyperTrie.prototype._ready = function (cb) {
 }
 
 HyperTrie.getMetadata = function (feed, cb) {
-  feed.get(0, { valueEncoding: Header }, (err, header) => {
+  feed.get(0, (err, msg) => {
     if (err) return cb(err)
-    return cb(null, header.metadata)
+    try {
+      var header = Header.decode(msg)
+      return cb(null, header.metadata)
```

Comment:
dont try-catch the cb itself. that’ll call the cb twice if the cb method fails

Reply:
:+1: